### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by Endmateria** [daipenger/Sound-Physics#3]
Server crashed and the crash report seems to be blaming SoundPhysics. 
I'm just pre-emptively posting this i.e., I haven't tried troubleshooting/isolating the conflicting mod yet... probably foamfix (though the mod does work fine with this mod setup when sound physics is loaded client side only).  Anyways, here's the crash report in the meantime... oh and thanks for forking and updating the mod!  Loving it.  https://gist.github.com/Endmateria/187bcc9092bdb80c9df87278efaa79c3#file-crash-2017-04-19_22-23-53-server-txt

**Comment by daipenger** [comment]
Why would you load it on a server anyways?
Given that you have closed this, I assume you have fixed it?

**Comment by Endmateria** [comment]
I loaded it on a server cause I was using serversync.... it downloads all the mods on your server to the client's computer.  It's an easy way of transferring mods i.e., load all the mods on the server, launch server, have all clients sync those mods to their computer, shut down server, remove all client side only mods from server... relaunch hahaha.  But yeah, I closed the issue because I just realized (after posting this) the difference between client/server side mods and the importance of NOT loading client side mods on the server side (mainly for performance reasons).  Sorry for the false alarm (ignorance on my part)... would've deleted the issue, but Github doesn't allow that, so I closed it instead.
